webapp/channels/src/components/threading/thread_viewer/thread_viewer.tsx

```diff
@@ -225,15 +224,14 @@ export default class ThreadViewer extends React.PureComponent<Props, State> {
                                 <DeferredThreadViewerVirt
                                     inputPlaceholder={this.props.inputPlaceholder}
                                     key={this.props.selected.id}
-                                    channel={this.props.channel}
+                                    channelId={this.props.channel.id}
```

Comment:
Good call. That'll definitely save someone from a headache at some point down the line.

If you want to though, you could make a plugin API version of this component which converts the `channel` prop to `channelId` so that you can make the web app version nicer without breaking any plugins like the AI plugin which are likely using this. See the `Textbox` in `plugins/textbox` for an example of that